packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewFlutterPlugin.java

```diff
@@ -120,7 +126,9 @@ public void onAttachedToEngine(@NonNull FlutterPluginBinding binding) {
         binding.getBinaryMessenger(),
         binding.getPlatformViewRegistry(),
         binding.getApplicationContext(),
-        null);
+        null,
+        new FlutterAssetManager.PluginBindingFlutterAssetManager(
+            binding.getApplicationContext().getAssets(), binding.getFlutterAssets()));
```

Comment:
If you're using the application context, doesn't this need updating in `updateContext`?

Reply:
I thought this would only be the case for the `Activity` context and the application context stays the same during the live time of the App.

Reply:
I'm still an Android n00b; sorry for the noise!